testdata/project-v3-addon-and-grafana/Makefile

```diff
@@ -68,14 +68,40 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
+# The build is supported with BuildKit enabled
+# If you disable this option then the manager image will be built using linux/amd64 by default
+# Alternatively, you can customize the Dockerfile by removing the --platform flag
+# However, in this case you will be unable to use the docker-buildx target to build with cross-compilation 
+# the manager to support multiple architectures
+export DOCKER_BUILDKIT=1
+
+# To get the system architecture to allow the build work if/when buildKit is disabled
+LOCALPLATFORM ?= linux/$(shell uname -m | sed 's/x86_64/amd64/' | sed 's/aarch64/arm64/')
```

Comment:
Would it be better to pull the local arch value from the GOENV values? Or can we not ensure that those will be set?

Reply:
Hi @jaypoulz, 

 > Would it be better to pull the local arch value from the GOENV values? 
No, because when we downstream it for SDK, SDK has plugins that work with Ansible/Helm/Java.

Therefore, for we ensure that all Makefiles (ideally) are == as much as possible looking for maintainability and supportability we cannot use Golang to check it out. See those other languages have no pre-requirement to have installed Go.

> Or can we not ensure that those will be set?
We have similar logic applied on SDK scaffolds and it has been working well.
I do not see why/when that would not be set under the official support platforms/operational systems

Reply:
It was changed to follow up the suggestion made so I am closing this one